lib/slimmer/railtie.rb

```diff
@@ -9,6 +9,8 @@ class Railtie < Rails::Railtie
       slimmer_config = slimmer_config.reverse_merge(app_name: app_name)
 
       app.middleware.use Slimmer::App, slimmer_config
+
+      I18n.backend = I18n::Backend::Chain.new(I18n.backend, Slimmer::I18nBackend.new)
```

Comment:
Not a railstie expert, but it looks like this will add the translation look up for any app that includes slimmer?

Components aren't automatically resolved in applications, they [need to be opted into](https://github.com/alphagov/government-frontend/blob/b9f350d6a5da7d3f9206b80c323a55203c36e883/app/controllers/application_controller.rb#L2).

Could the the loading of translation look behaviour be done [in the same way](https://github.com/alphagov/slimmer/blob/0d9d77a8bc658602a117e34d8eb34cae26d85643/lib/slimmer/shared_templates.rb#L8), so we're not looking up templates when not using components?

Longer term components should be enabled by default, as all apps should be using them, but in the mean time we should keep the component and translation lookup behaviour the same. Partially because we have some [fragility in calls to static ](https://github.com/alphagov/slimmer/issues/149) and i'd like to avoid making them if we don't need them.